app/src/main/java/me/saket/dank/ui/submission/SubmissionPageLayout.java

```diff
@@ -278,6 +282,32 @@ public void smoothScrollToPosition(RecyclerView recyclerView, RecyclerView.State
         linearSmoothScroller.setTargetPosition(position);
         startSmoothScroll(linearSmoothScroller);
       }
+
+      // Workaround to keep arrow keys navigation working
```

Comment:
I haven't actually tested this since I've got no working OTG cable. Arrow navigation aside, two methods below can be effectively replaced by just `return false`

Reply:
Do you think that this is something we should worry about? I've actually never used arrow navigation myself so have no idea how that would affect it.

Reply:
Well, keyboards and stuff. Dawn already uses some rather heavy UI mods so I wouldn't be surprised if it's already broken. Well, then I just cover these methods with false?

Reply:
Let’s keep it. Seems good enough as is for me.